source/enumerate-indexes.rst

```diff
@@ -369,16 +380,16 @@ The shell implements the first algorithm for falling back if the
 Version History
 ===============
 
-Version 0.3 Changes
-
-    - Put preferred method name for listing indexes with a cursor as return
-      value.
-
-Version 0.2 Changes
+0.3.1
```

Comment:
While we're here could you "git annotate" and put dates on the changes, the same as we have in other specs?

Reply:
I've done it for 0.4 and 0.3, but for earlier versions that information is only kept in the Google Doc I believe. "git log" doesn't show any commits besides the initial conversion to RST and the preferred name.